docs/whats-new/additional-resources.md

```diff
@@ -0,0 +1,45 @@
+# Additional resources for service changes
```

Comment:
hmmm..I don't think we should have this section is separated out as additional resources here in what's new..

2 things:
1. the content of this is a guide for only tenant admins. so it should only be in tenant guide
2. this guide is not a temporarily content. it's something useful like a blog. like the section in concepts: https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/#/concepts/oauth2. I will prefer such guides be treated as guides and not as additional resources that can only be found in what's new. content in what's news should point to the guides residing in more permanent locations